config/locales/en.yml

```diff
@@ -502,6 +505,8 @@ news:
       Our press releases will give you the key facts about funding announcements, new grants or research, special events, launches, which charities have been awarded funding, innovative projects, best practice in the sector and more.
     blog: >
       Looking for colour, context and perspective about the work of The National Lottery Community Fund, its impact on communities and topics currently in the news? It’s all here. This blog will explain, explore, showcase and examine the ways in which civil society, and as a result the sector, is changing. Read about what people in communities across the UK and sector thought leaders have to say and form your own opinion.
+    people-stories: >
```

Comment:
This is `peopleStories` earlier but `people-stories` here, is camel cased for both possible?

Reply:
Ideally the other way around, as I'm using the lowercase one (supplied via URL params) as a variable everywhere else.